.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/app.log
```

```suggestion
logs/*
```

I think it's better to exclude the whole logs folder, so if you will add new logger file in the future, you won't need to add changes here